specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json

```diff
@@ -1643,6 +1691,10 @@
           "$ref": "#/definitions/PrivateEndpointConnectionProperties",
           "x-ms-client-flatten": true,
           "description": "Resource properties."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Modified whenever the user or NRP changes the state of PE connection."
```

Comment:
Since this is exposed to public, I recommend a description such as `Modified whenever there is a change in the state of private endpoint connection`.

Reply:
Sure.